src/components/ContentBox/ContentBox.tsx

```diff
@@ -18,6 +19,7 @@ interface IContentBoxProps {
   title?: string;
   contentHeight?: string;
   description?: IDescription;
+  isResponsive?: boolean;
```

Comment:
Add some TSDoc explaining when it should be used.

Reply:
Added accordingly